python/setup.py.i

```diff
@@ -2,22 +2,43 @@ from setuptools import setup, find_packages
 
 import sys
 if sys.version_info < (3,0):
-  sys.exit('Sorry, Python < 3.0 is not supported')
+  sys.exit("Sorry, Python < 3.0 is not supported")
+
+try:
+    from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+    class bdist_wheel(_bdist_wheel):
+        def finalize_options(self):
+            _bdist_wheel.finalize_options(self)
+            self.root_is_pure = False
+except ImportError:
+    bdist_wheel = None
 
 requirements = [
-    'numpy',
-    #'tqdm',
-    #'requests',
-    #'portalocker',
-    #'opencv-python'
+    "numpy",
+    #"tqdm",
+    #"requests",
+    #"portalocker",
+    #"opencv-python"
 ]
 
 setup(
-  name          = 'ncnn',
-  version       = '${PACKAGE_VERSION}',
-  url           = 'https://github.com/Tencent/ncnn',
-  packages      = find_packages(),
-  package_dir   = {'': '.'},
-  package_data  = {'ncnn': ['ncnn${PYTHON_MODULE_PREFIX}${PYTHON_MODULE_EXTENSION}']},
-  install_requires = requirements
+  name              = "ncnn",
+  version           = "${PACKAGE_VERSION}",
+  author            = "nihui",
+  author_email      = "nihuini@tencent.com",
+  maintainer        = "caishanli",
+  maintainer_email  = "caishanli25@gmail.com",
+  description       = "ncnn is a high-performance neural network inference framework optimized for the mobile platform",
+  url               = "https://github.com/Tencent/ncnn",
+  classifiers       = [
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
```

Comment:
bsd 3-clause ?

Reply:
this is copied from xiaohanxiao's kears2ncnn......
```License :: OSI Approved :: BSD License```
is this correct?

Reply:
all license is in [classifiers](https://pypi.org/classifiers/)

Reply:
BSD License is fine